tests/xcimporter/test_align.py

```diff
@@ -1,15 +1,21 @@
+import shutil
```

Comment:
We will need to adapt the tests in the future to check the content of the output. Another suggestion: dump the contents of all of the files you've added to a json file/object. If an object, you can use it in the test itself and write it to a temp file or using ``stringIO`` (better). Some examples here: https://stackoverflow.com/questions/3942820/how-to-do-unit-testing-of-functions-writing-files-using-python-unittest